test/assert.js

```diff
@@ -120,6 +120,10 @@ test('.not()', function (t) {
 });
 
 test('.deepEqual()', function (t) {
+	t.throws(function () {
+		assert.deepEqual({a: false}, {a: 0});
+	});
```

Comment:
Sure, that, and maybe some tests from https://github.com/sotojuan/not-so-shallow Just to make sure we never regress, even if we were to replace your lib with something else any time in the future (hopefully we won't though).